src/commons/utils/nimon_resource_util.py

```diff
@@ -49,17 +50,17 @@ def monitor_resource_utilisation(action: str):
             host, user, passwd = node["hostname"], node["username"], node["password"]
             server_nodes.extend(host)
             cluster_obj = RemoteHost(host, user, passwd)
-            resp = cluster_obj.execute_command(cm_cmd.K8S_WORKER_NODES)
+            resp = cluster_obj.execute_command(cm_cmd.KUBECTL_GET_WORKERS_NAME)
             LOGGER.debug("response is: %s", str(resp))
             worker_node = resp[1].strip().split("\n")[1:]
             LOGGER.info("worker nodes: %s", str(worker_node))
             server_nodes.extend(worker_node)
     if action == "start":
```

Comment:
May we keep these logic in different methods

Reply:
@akshaym99 will be taking care this in his task.